src/redux/Books/books.js

```diff
@@ -0,0 +1,18 @@
+import { createReducer } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const bookReducer = createReducer(initialState, (construct) => {
+  construct.addCase('CREATE_BOOK', (state, action) => {
+    state.books.push(action.payload);
+  });
+  construct.addCase('REMOVE_BOOK', (state, action) => {
```

Comment:
- [ ] Kindly use the ducks pattern as mentioned in the project requirements. `CREATE_BOOK` and `REMOVE_BOOK` should be something like `bookstore/books/CREATE_BOOK`... (same for `CHECK_STATUS` in categories) 👍